retriever/engines/postgres.py

```diff
@@ -17,21 +18,30 @@ class engine(Engine):
         "bool": "boolean",
     }
     max_int = 2147483647
+    try:
```

Comment:
let us avoid the try block. If we know the user and we check the path we shall not need this. Else we need to give a specific reason or comment for the except IOError. I may be missing something let me know